### PR TITLE
add vue assistant

### PR DESCRIPTION
**Pull request by dosstx**

### CODE REVIEW

**Comment by dosstx** [comment]
@lgrammel  I tried to submit this PR but I get errors during pipeline process. I did follow the Next examples for Nuxt. Can you take a look and get this merged? @danielroe Can you take a look at this Nuxt implementation of the `useAssistant` composable I made for the AI SDK?

**Comment by danielroe** [comment]
It looks like you need to update the lock file. Ensure you have enabled corepack with `corepack enable`.

File: examples/nuxt-openai/server/api/assistant.ts [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+  'living room': 21,
+  kitchen: 22,
+  bathroom: 23,
+};
+
+export default defineEventHandler(async event => {
+  const config = useRuntimeConfig(event);
+
+  if (!config.openaiApiKey) {
+    throw new Error('OPENAI_API_KEY is not set');
```
Comment (danielroe) [inline examples/nuxt-openai/server/api/assistant.ts:16]: Should we have NUXT_ prefix in these logs?

Comment (dosstx) [reply examples/nuxt-openai/server/api/assistant.ts:16]: Yes

**Comment by dosstx** [comment]
Made a new one at: https://github.com/vercel/ai/pull/1692

**Pull request closed without merging**